widget.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "List (no images)",
   "package": "com.fliplet.simple-list",
-  "version": "1.0.0",
```

Comment:
sure about this?

Reply:
Yep. I need this to be a small increase otherwise it can break custom CSS added by CS or clients